src/docs/static-site-generation/meta.md

```diff
@@ -5,15 +5,21 @@ url: /docs/static-site-generation-meta-tags
 contributors:
   - mlynch
   - adamdbradley
+  - rwaskiewicz
 ---
 
 # SEO Meta Tags and Static Site Generation
 
 Web Apps need to list detailed meta information about content in order to maximize SEO and provide good social media embed experiences.
 
-One of the benefits to Stencil's prerendering is that most DOM apis are available in the NodeJS environment too. For example, to set the document title, simply run `document.title = "Page Title"`. Or meta tags can be added and updated no differently than using the usual DOM APIs found in the browser, such as `document.head` and `document.createElement('meta')`. This mean your components runtime may already be able to do much of the custom work throughout prerendering.
+One of the benefits to Stencil's prerendering is that most DOM apis are available in the NodeJS environment too.
+For example, to set the document title, simply run `document.title = "Page Title"`.
+Or meta tags can be added and updated no differently than using the usual DOM APIs found in the browser, such as `document.head` and `document.createElement('meta')`.
```

Comment:
Possible rewording section (I know this was already here, but figure why not tweak a little if it's kinda odd)

```suggestion
Similarly, meta tags can be set using standard DOM APIs as found in the browser, such as `document.head` and `document.createElement('meta')`.
```